firefox-ios/nimbus-features/creditCardAutofillFeature.yaml

```diff
@@ -1,17 +0,0 @@
-features:
-  credit-card-autofill:
-    description: This property defines the credit card autofill feature
```

Comment:
@razvanlitianu any reason why we are removing CC feature? 🤔

Reply:
I see that you added [autofillFeature.yaml](https://github.com/mozilla-mobile/firefox-ios/pull/18866/files#diff-4ce029336fd9e9d18f5f74ae216bf35d302e2b205d4ab894d4b6848eeb6d283b) so I am guessing its taken care there

Reply:
Bad diffing, the file was renamed to accommodate all autofill experiments

Reply:
ha, gotcha